packages/vue3/src/components/Tooltip/Tooltip.vue

```diff
@@ -16,84 +76,36 @@ defineProps<{
 
 <style scoped lang="scss">
 .tooltip {
-  position: relative;
   display: inline-block;
+  position: relative;
 
   .tooltip-trigger {
     visibility: hidden;
-    opacity: 0;
-    position: absolute;
-    z-index: 99;
+    position: fixed;
+    z-index: 9999999999;
     width: max-content;
-    max-width: 300px;
+    max-width: 200px;
     padding: 8px 12px;
-    transition: all 0.2s ease-in-out;
+    transition: opacity 0.3s ease-in-out;
+    transition-delay: 0.1s;
     border-radius: 4px;
+    opacity: 0;
     background-color: #000;
```

Comment:
Use variables instead

```suggestion
    background-color: var(--base-white);
```